src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs

```diff
@@ -226,5 +226,13 @@ internal IEnumerable<SecurityScheme> GetSchemesOrAnonymous()
             }
         }
     }
+
+    internal partial class OperationGroup
+    {
+        public override string ToString()
```

Comment:
I prefer the shorter version:

```suggestion
        public override string ToString() => $"OperationGroup(Key: {Key})";
```

But longer is fine as well.
```